osu.Server.Queues.ScoreStatisticsProcessor/Models/SoloScore.cs

```diff
@@ -17,43 +18,72 @@ public class SoloScore
         [ExplicitKey]
         public ulong id { get; set; }
 
-        public int user_id { get; set; }
+        public uint user_id { get; set; }
 
-        public int beatmap_id { get; set; }
+        public uint beatmap_id { get; set; }
 
-        public int ruleset_id { get; set; }
+        public ushort ruleset_id { get; set; }
+
+        public bool has_replay { get; set; }
+        public bool preserve { get; set; }
+        public bool ranked { get; set; } = true;
+
+        public ScoreRank rank { get; set; }
+
+        public bool passed { get; set; } = true;
+
+        public float accuracy { get; set; }
+
+        public uint max_combo { get; set; }
+
+        public uint total_score { get; set; }
+
+        public SoloScoreData ScoreData = new SoloScoreData();
 
         public string data
         {
-            get => JsonConvert.SerializeObject(ScoreInfo);
+            get => JsonConvert.SerializeObject(ScoreData);
             set
             {
-                var scoreInfo = JsonConvert.DeserializeObject<SoloScoreInfo>(value);
-
-                if (scoreInfo == null)
-                    return;
-
-                ScoreInfo = scoreInfo;
-
-                ScoreInfo.ID = id;
-                ScoreInfo.BeatmapID = beatmap_id;
-                ScoreInfo.UserID = user_id;
-                ScoreInfo.RulesetID = ruleset_id;
+                var soloScoreData = JsonConvert.DeserializeObject<SoloScoreData>(value);
+                if (soloScoreData != null)
+                    ScoreData = soloScoreData;
             }
         }
 
-        [JsonIgnore]
-        public bool preserve { get; set; }
+        public double? pp { get; set; }
 
-        [JsonIgnore]
-        public bool has_replay { get; set; }
+        public ulong? legacy_score_id { get; set; }
+        public uint? legacy_total_score { get; set; }
 
-        [JsonIgnore]
-        public SoloScoreInfo ScoreInfo = new SoloScoreInfo();
-
-        [JsonIgnore]
-        public DateTimeOffset created_at { get; set; }
+        public DateTimeOffset? started_at { get; set; }
+        public DateTimeOffset ended_at { get; set; }
 
         public override string ToString() => $"score_id: {id} user_id: {user_id}";
+
+        public ushort? build_id { get; set; }
```

Comment:
I don't see this on the table definition, is this from join context or something?

Reply:
need to update table definition :D